integration-libs/opf/global-functions/core/facade/opf-global-functions.service.ts

```diff
@@ -39,17 +40,24 @@ export class OpfGlobalFunctionsService implements OpfGlobalFunctionsFacade {
   protected ngZone = inject(NgZone);
   protected opfPaymentFacade = inject(OpfPaymentFacade);
   protected launchDialogService = inject(LaunchDialogService);
-
+  protected opfCtaFacade = inject(OpfCtaFacade);
   protected loaderSpinnerCpntRef: void | Observable<
     ComponentRef<any> | undefined
   >;
+  protected _readyForScriptEvent: Subject<string> = new Subject();
+  readyForScriptEvent$: Observable<string> =
+    this._readyForScriptEvent.asObservable();
 
   registerGlobalFunctions({
     domain,
     paymentSessionId,
     vcr,
     paramsMap,
   }: GlobalFunctionsInput): void {
+    // SSR not supported
```

Comment:
[MINOR]
Let's explain why it's not supported
a) it's not suported YET (but we plan to add a support)?
vs
b) it's not supported by design - in SSR we don't want to register global functions, becasue they are meant only for the user interactions in the browser

Note: analogical remark applies to other place in this file